### PR TITLE
Abacus 2.0: Subsequent empty Order field blocks attribute creation.

### PR DESCRIPTION
**Pull request by ysizarie-virtru**
### Ticket: https://virtru.atlassian.net/browse/PLAT-1868

### CODE REVIEW

**Review by ivanovSPvirtru — DISMISSED** [review]

File: src/containers/Attributes/components/CreateAttributeForm.tsx [hunk 1]
```diff
@@ -75,11 +75,11 @@ const CreateAttributeForm: FC<Props> = (props) => {
 
               return (
                 <Item
-                  required
+                  required={!Number(field.key)}
```
Comment (obidenko) [inline src/containers/Attributes/components/CreateAttributeForm.tsx:78]: Why require depends on "Not a number or 0" ?

File: src/containers/Attributes/CreateAttribute.tsx [hunk 2]
```diff
@@ -48,10 +48,16 @@ const CreateAttribute: FC<Props> = (props) => {
   );
 
   const handleCreateAttribute = (values: CreateAttributeValues) => {
+    const formData = {
+      ...values,
+      order: values.order.filter(Boolean),
```
Comment (obidenko) [inline src/containers/Attributes/CreateAttribute.tsx:53]: Add, pls, comment-link to the trick for future generations)

File: src/containers/Attributes/components/CreateAttributeForm.tsx (hunk 1 above)
Comment (obidenko) [reply src/containers/Attributes/components/CreateAttributeForm.tsx:78]: Could we move it to const?

Comment (ysizarie-virtru) [reply src/containers/Attributes/components/CreateAttributeForm.tsx:78]: We need at least one order value in array. First one will have key 0 (required), every following will be optional

**Review by obidenko — APPROVED** [review]

**Pull request merged**